popup.js

```diff
@@ -0,0 +1,103 @@
+    lang3: 'JavaScript',
+    link1: 'https://github.com/kal2944/mobile-version-skeleton',
+    link2: 'https://github.com/kal2944/mobile-version-skeleton',
+  },
+
+  {
+    id: 1,
+    projTitle: 'Multi Post Stories',
+    projDetails: 'Canopy &nbsp; &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2015',
+    projPic: './images/card4.svg',
+    proInfo: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    lang1: 'HTML',
+    lang2: 'CSS',
+    lang3: 'JavaScript',
+    link1: 'https://github.com/kal2944/mobile-version-skeleton',
+    link2: 'https://github.com/kal2944/mobile-version-skeleton',
+  },
+  {
+    id: 2,
+    projTitle: 'Tonic',
+    projDetails: 'Canopy &nbsp; &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2015',
+    projPic: './images/mobil-card1.svg',
+    proInfo: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    lang1: 'HTML',
+    lang2: 'CSS',
+    lang3: 'JavaScript',
+    link1: 'https://github.com/kal2944/mobile-version-skeleton',
+    link2: 'https://github.com/kal2944/mobile-version-skeleton',
+  },
+
+  {
+    id: 3,
+    projTitle: 'Multi Post Stories',
+    projDetails: 'Canopy &nbsp; &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2015',
+    projPic: './images/card2.svg',
+    proInfo: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    lang1: 'HTML',
+    lang2: 'CSS',
+    lang3: 'JavaScript',
+    link1: 'https://github.com/kal2944/mobile-version-skeleton',
+    link2: 'https://github.com/kal2944/mobile-version-skeleton',
+  },
+];
+
+function fillPopUp(arr, num) {
+  const proj = arr[num];
+  projTitle.textContent = proj.projTitle;
+  projDetails.innerHTML = proj.projDetails;
+  projPic.src = proj.projPic;
+  proInfo.textContent = proj.proInfo;
+  lang1.textContent = proj.lang1;
+  lang2.textContent = proj.lang2;
+  lang3.textContent = proj.lang3;
+  proLink1.setAttribute('href', proj.link1);
+  proLink2.setAttribute('href', proj.link2);
+}
+
+seeButton.addEventListener('click', () => {
+  popUp.classList.toggle('active');
+  fillPopUp(projArr, 0);
+});
+
+seeButton1.addEventListener('click', () => {
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you improve the again the styling of the popup, both on Mobile and Desktop, to match to the the Figma template(e.g.: update the font(size && weight) like on the project's tile, update the the **See Live** and **See Source** styling,...) Check other details from Figma to make your Popup modal sticks to expected template.

| Figma Design | Your Design |
| -------------------- | ------------------- |
| ![Screenshot from 2022-07-20 12-25-23](https://user-images.githubusercontent.com/83042742/181935479-519709c0-4945-47fb-8583-5ade70ed6b83.png) | ![Screenshot from 2022-07-30 19-43-55](https://user-images.githubusercontent.com/83042742/181935496-5ac9a37a-8746-4ded-972d-728d72a73dc5.png) |